app/assets/javascripts/short-form/templates/b2-contact.html.slim

```diff
@@ -4,16 +4,16 @@
       | {{'B2_CONTACT.TITLE' | translate:applicantFirstName()}}
 
   .app-inner.inset.border-bottom
-    .form-group.has-option ng-class="{ error: $scope.requiredContactInformationMissing() }"
+    .form-group.has-option ng-class="{ error: inputInvalid('phone') }"
```

Comment:
🙌 <-- magic fingers. Really though, I realized that everything came together nicely by just using the `inputInvalid` and just letting the alertBox show up under the right conditions (e.g. the form is submitted and invalid _and_ `requiredContactInformationMissing`). The reason the `ng-class=error` works is because we are saying "this field is required IF required contact info is missing" -- therefore it's the `ng-required` that kicks in to trigger the `inputInvalid` by saying "due to the missing info conditions, this form is now required"